roles/openshift_master_facts/tasks/main.yml

```diff
@@ -0,0 +1,145 @@
+---
+- name: Check for RPM generated config marker file .config_managed
+  stat:
+    path: /etc/origin/.config_managed
+  register: rpmgenerated_config
+
+- name: Remove RPM generated config files if present
+  file:
+    path: "/etc/origin/{{ item }}"
+    state: absent
+  when: rpmgenerated_config.stat.exists == true and deployment_type in ['openshift-enterprise', 'atomic-enterprise']
+  with_items:
+  - master
+  - node
+  - .config_managed
+
+- set_fact:
+    openshift_master_pod_eviction_timeout: "{{ lookup('oo_option', 'openshift_master_pod_eviction_timeout') | default(none, true) }}"
+  when: openshift_master_pod_eviction_timeout is not defined
+
+- set_fact:
+    openshift_master_etcd_port: "{{ (etcd_client_port | default('2379')) if (groups.oo_etcd_to_config is defined and groups.oo_etcd_to_config) else none }}"
+    openshift_master_etcd_hosts: "{{ hostvars
+                                     | oo_select_keys(groups['oo_etcd_to_config']
+                                                      | default([]))
+                                     | oo_collect('openshift.common.hostname')
+                                     | default(none, true) }}"
+
+- set_fact:
+    openshift_master_debug_level: "{{ lookup('oo_option', 'openshift_master_debug_level') | default(openshift.common.debug_level, true) }}"
+  when: openshift_master_debug_level is not defined
```

Comment:
probably want to move this down into the openshift_facts task as well to avoid doing this in a separate task.